### PR TITLE
feat(telecom.pack): add multi otp choice on move and migrate

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-583
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Add multi otp choice on move and migrate offers

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/building-details/migration-building-details.html [hunk 1]
```diff
@@ -110,6 +110,24 @@ <h3 data-translate="telecom_pack_migration_pto_title"></h3>
...
+                            model="$ctrl.model.pto"
+                            value="$ctrl.FIBER_PTO.FIBER_PTO_MULTI_OTP"
+                        >
+                            <span
+                                data-translate="telecom_pack_migration_pto_multi_otp"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                oui-popover="{{'telecom_pack_migration_pto_multi_otp_description' | translate}}"
```
Comment (antonymarion) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/building-details/migration-building-details.html:126]: data-*

File: packages/manager/apps/telecom/src/app/telecom/pack/move/building-details/move-building-details.html [hunk 2]
```diff
@@ -101,8 +101,24 @@ <h3 data-translate="pack_move_pto_title"></h3>
...
+                    <oui-radio
+                        name="radioPto"
+                        model="$ctrl.model.pto"
+                        value="$ctrl.FIBER_PTO.FIBER_PTO_MULTI_OTP"
+                    >
+                        <span data-translate="pack_move_pto_multi_otp"></span>
+                        <button
+                            type="button"
+                            class="oui-popover-button"
+                            oui-popover="{{'pack_move_pto_multi_otp_description' | translate}}"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/building-details/move-building-details.html:115]:
```suggestion
                            data-oui-popover="{{'pack_move_pto_multi_otp_description' | translate}}"
```

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js [hunk 3]
```diff
@@ -169,6 +173,32 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
       });
     }
 
+    // Installation type post params
+    if (migrationProcess.selectedOffer.pto) {
```
Comment (antonymarion) [inline packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js:177]: Maybe replace switch case by a dictionnary here => will save ~20 lines

Comment (Steffy29) [reply packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js:177]: Hello @antonymarion, would you have an example on how to do it? it's a good idea.

Comment (ghyenne) [reply packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js:177]: You can put a new object in constant file:
```js
const MY_OBJECT = {
  [FIBER_PTO.FIBER_PTO_YES]: INSTALL_TYPE.activate,
  [FIBER_PTO.FIBER_PTO_YES_BUT_NOT_KNOWN]: INSTALL_TYPE.activate_undefined,
  [FIBER_PTO.FIBER_PTO_MULTI_OTP]: INSTALL_TYPE.multi_otp,
  DEFAULT: INSTALL_TYPE.create,
};
```
And use it on controller:

```js
if (migrationProcess.selectedOffer.pto) {
  assign(postParams, {
    installationType:
      MY_OBJECT[migrationProcess.selectedOffer.selectedPto] ||
      MY_OBJECT.DEFAULT,
  });
}
```

File: packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.controller.js [hunk 4]
```diff
@@ -247,6 +252,30 @@ export default class MoveResumeCtrl {
...
+            assign(moveData, {
+              installationType: INSTALLATION_TYPE.multiOtp,
+            });
+            break;
+          default:
+            assign(moveData, {
+              installationType: INSTALLATION_TYPE.create,
+            });
+            break;
+        }
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.controller.js:278]: same comment https://github.com/ovh/manager/pull/11795#discussion_r1622431669

**Review by antonymarion — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Comment by AdilCHAGRAOUI** [comment]
release/components-w24

**Pull request merged**